hpx/parallel/algorithms/inclusive_scan.hpp

```diff
@@ -184,11 +185,12 @@ namespace hpx { namespace parallel { inline namespace v1
         }
 
         // forward declare the segmented version of this algorithm
-        template <typename ExPolicy, typename InIter, typename OutIter, typename T,
-            typename Op>
+        template <typename ExPolicy, typename InIter, typename OutIter,
+            typename T, typename Op, typename Conv = util::projection_identity>
         static typename util::detail::algorithm_result<ExPolicy, OutIter>::type
-        inclusive_scan_(ExPolicy&& policy, InIter first, InIter last, OutIter dest,
-            T const& init, Op && op, std::true_type);
+        inclusive_scan_(ExPolicy&& policy, InIter first, InIter last, OutIter
+            dest, T const& init, Op && op, std::true_type, Conv && conv =
+            Conv());
         /// \endcond
```

Comment:
Please try not to separate argument type and argument name onto different lines.